pages/api/events.ts

```diff
@@ -1,37 +1,58 @@
 import Analytics from "@rudderstack/rudder-sdk-node";
 import { apiObject } from "rudder-sdk-js";
 // we need the batch endpoint of the Rudder server you are running
-const client = new Analytics(process.env.RUDDERSTACK_SERVER_WRITE_KEY!, { dataPlaneUrl: process.env.RUDDERSTACK_SERVER_DATA_PLANE_URL });
-
-const rudderStackEvents = {
-    identify: (userId: string, name: string, email: string, githubId: string, role: string, anonymousId: string) => { // Anonymous Id is set in local storage as soon as a user lands on the webiste.
-        console.log("identify");
-        client.identify({
-            userId: userId,
-            anonymousId: anonymousId,
-            traits: {
-                name: name,
-                email: email,
-                githubId: githubId,
-                role: role
+
+    const client = new Analytics(process.env.RUDDERSTACK_SERVER_WRITE_KEY!, { dataPlaneUrl: process.env.RUDDERSTACK_SERVER_DATA_PLANE_URL });
+
+
```

Comment:
Instead, you can have a condition on the definition of the `client` object. I propose that we create mock `identify` and `track` functions for the client object which just print to the console when the NODE_ENV is `development`.

Reply:
Something like this:

```suggestion
    const client = (process.env.NODE_ENV === 'development') ? {
        identify: (event_properties: <use type given by RudderStack>) => console.info(event_properties),
        track: (event_properties: <again use RudderStack type>) => console.info(event_properties)
    } 
    : new Analytics(process.env.RUDDERSTACK_SERVER_WRITE_KEY!, { dataPlaneUrl: process.env.RUDDERSTACK_SERVER_DATA_PLANE_URL });
```